app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -350,13 +353,55 @@ def populate_hardware_hash_with_series_attributes(hardware_hash, instance, serie
         hardware_hash[:memory_mb]       = series[:memory] / 1.megabyte
         hardware_hash[:disk_capacity]   = series[:root_disk_size] + series[:swap_disk_size]
 
```

Comment:
Is the disk capacity correct? I do not think we should get this value from the series.

Reply:
I would have to know the difference between size and disk_capacity. And that's not part of the disks table. It may actually be correct.